internal/handler/order.go

```diff
@@ -0,0 +1,65 @@
+package handler
+
+import (
+	"errors"
+	"github.com/Azzonya/gophermart/internal/entities"
+	"github.com/Azzonya/gophermart/internal/errs"
+	"github.com/gin-gonic/gin"
+	"io"
+	"net/http"
+)
+
+func (u *UserHandlers) UploadOrder(c *gin.Context) {
+	// Реализация загрузки номера заказа
+	body, err := io.ReadAll(c.Request.Body)
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, gin.H{"error": "Error reading request body"})
+		return
+	}
+
+	orderNumber := string(body)
+	userID, _ := u.userUsecase.GetUserIDFromCookieU(c)
+
+	err = u.orderUsecase.Create(c.Request.Context(), &entities.Order{
+		OrderNumber: orderNumber,
+		Status:      entities.OrderStatusNew,
+		UserID:      userID,
+	})
```

Comment:
это все и есть юзкейс. Мы это уже обсуждали - юзейс это целый набор инструкций который выполняет полностью какой-то кусок логики - Тут в ручке должно быть только 
```suggestion
	err = u.orderUsecase.CreateOrder(c.Request.Context(), &entities.Order{
		OrderNumber: orderNumber,
		Status:      entities.OrderStatusNew,
	})
```